src/components/SwitchOnOff.js

```diff
@@ -28,11 +28,23 @@ export default class SwitchOnOff extends Component {
 
         /*The div switch-container has an event listener that will triggle the toggle method if clicked
         Based on the trigger for toggle, h1 tag and background color of box will change*/
+        //The lightBulb component is rendered with the passed on state property lightSwitch
+        //The text "I" is meant for on and the "O" is meant for off.
         return (
-            <div onClick={this.handleToggleChange} className="switch-container">
-                <h1>Light Bulb Challenge</h1>
-                <div className={lightSwitch ? "switch-box switch-on" : "switch-box switch-off"}>
-                    <h1 id="switch-label">{lightSwitch ? "On" : "Off"}</h1>
+            <div className="switchOnOff-container">
+                <h1 id="switchOnOff-title">Edison Light</h1>
+                <LightBulb lightOn={this.state.lightSwitch} />
+
+                <div className="wrapperTwoForButton">
+                    <div className="wrapperOneForButton">
+                        <div onClick={this.handleToggleChange} >
+                            <div className="switch-box">
+                                <div className="switch-label">
+                                    <h1 id="icon">{lightSwitch ? "I" : "O"}</h1>
```

Comment:
I love the animation here, but i think instead of the "O" you should do "〇" which is a circle so I think when it spins it won't look wobbly, though the wobbly has a nice affect too since it'll be noticeable. 🤷